ERCS/erc-5559.md

```diff
@@ -1,471 +1,449 @@
 ---
 eip: 5559
-title: "Cross Chain Write Deferral Protocol"
-description: The cross chain write deferral protocol provides a mechanism to defer the storage & resolution of mutations to off-chain handlers
-author: Paul Gauvreau (@0xpaulio), Nick Johnson (@arachnid)
-discussions-to: https://ethereum-magicians.org/t/eip-cross-chain-write-deferral-protocol/10576
-status: Stagnant
+title: "Cross-chain Write Deferral Protocol"
+description: The cross-chain write deferral protocol provides a mechanism to replace L1 storage with L2 and databases through off-chain handlers
```

Comment:
why do you replace `defer` to `replace` when the title uses the word `Deferral`?

Reply:
Also do you need to change the title from `Cross Chain` to `Cross-chain`? It looks inconsistent

Reply:
> do you need to change the title from `Cross Chain` to `Cross-chain`

Correct terminology is `off-chain`, `on-chain` and `cross-chain`, not `off chain`, `on chain` or `cross chain`

Reply:
https://chain.link/cross-chain is using as `Cross-chain` so you are right